### ISSUE TITLE
using system call to run external programs eg matlab

### ISSUE THREAD

**Issue by zrxing** [stephens999/dscr#24]
when using system() R proceeds to next command once external program has opened (ie completion of system() command equals to opening the program, not when the program finishes running)

**Comment by stephens999** [comment]
As we discussed, this appears to be a windows issue, correct?

**Comment by zrxing** [comment]
That's right. I guess a workaround would be to just check the OS of the system and add the option to force R to wait if it's windows.
